docs/source/examples/outbrains.ipynb

```diff
@@ -1 +0,0 @@
-../../../examples/wnd_outbrain/E2E-NVT.ipynb
```

Comment:
I don't think we can delete this - did you mean to rename from outbrains.ipynb -> outbrain.ipynb?

Reply:
Yes.

Reply:
@benfred cant we remove E2E-NVT.ipynb and add a new notebook named `Outbrain.ipynb`? I will create a PR for outbrain notebook, and I want to name the notebook as `Outbrain.ipynb`.

Reply:
yes - we can change the filename of the notebook

We can't delete this symlink entirely though, it's needed to render the outbrain notebook in the docs. Once you've changed the filename can you add a new symlink to the file (also changing the symlinks name from outbrains -> outbrain?)